rfcs/text/0000-rfc-resource-tagging.md

```diff
@@ -0,0 +1,200 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# IaaS Resource Tagging for and Cost Monitoring and Auditing
```

Comment:
I suggest changing "IaaS" --> "Infrastructure as a Service (IaaS)" to make the abbreviation clearer

Reply:
implemented